modules/hardware/nvidia-jetson-orin/virtualization/passthrough/uarti-net-vm/default.nix

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  cfg = config.ghaf.hardware.nvidia.passthroughs.uarti_net_vm;
+in {
+  options.ghaf.hardware.nvidia.passthroughs.uarti_net_vm.enable = lib.mkOption {
+    type = lib.types.bool;
+    default = false;
+    description = "Enable UARTI passthrough on Nvidia Orin to the Net-VM";
+  };
+
+  config = lib.mkIf cfg.enable {
+    ghaf.hardware.nvidia.virtualization.enable = true;
+
+    ghaf.virtualization.microvm.netvm.extraModules = [
+      {
+        microvm.qemu.extraArgs = [
+          # Add custom dtb to Net-VM with 31d0000.serial in platform devices
+          "-dtb"
+          "${config.hardware.deviceTree.package}/tegra234-p3701-ghaf-nev-vm.dtb"
```

Comment:
Should this be net-vm instead of nev-vm?

Reply:
The problem with Robot Framework was that it uses the netvm serial console to qemu for testing if the vm boots correctly. Due the console was redirected to the UART passthrough the test failed. Due the Linux kernel is not able to use the console in two UART ports of the same kind, a virtual pci-serial console was used as qemu console output. For this, the regular console was disabled in microvm by a PR that I requested and that it was merged and tested https://github.com/astro/microvm.nix/pull/194

Also, typos in nev->net were fixed.